<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartcalcView</class>
 <widget class="QWidget" name="SmartcalcView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">background-color: darkcyan;</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>626</width>
        <height>326</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTextEdit" name="textEdit">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(192, 191, 188);</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_55">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_53">
               <item>
                <widget class="QPushButton" name="pushBuildGraph">
                 <property name="styleSheet">
                  <string notr="true">background-color: green</string>
                 </property>
                 <property name="text">
                  <string>Build a graph</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_2">
                 <property name="maximumSize">
                  <size>
                   <width>421</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(192, 191, 188);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="whatsThis">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border-top-color: rgb(224, 27, 36);</string>
                 </property>
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;X =&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::LinksAccessibleByMouse</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_56">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QPushButton" name="Button_asin">
                 <property name="text">
                  <string>asin</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_sin">
                 <property name="text">
                  <string>sin</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_mod">
                 <property name="styleSheet">
                  <string notr="true">background-color: yellow;
color: rgb(224, 27, 36);</string>
                 </property>
                 <property name="text">
                  <string>mod</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_div">
                 <property name="styleSheet">
                  <string notr="true">background-color: yellow;
color: rgb(224, 27, 36);</string>
                 </property>
                 <property name="text">
                  <string>/</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QPushButton" name="Button_acos">
                 <property name="text">
                  <string>acos</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_cos">
                 <property name="text">
                  <string>cos</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_pow">
                 <property name="styleSheet">
                  <string notr="true">background-color: yellow;
color: rgb(224, 27, 36);</string>
                 </property>
                 <property name="text">
                  <string>^</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_mul">
                 <property name="styleSheet">
                  <string notr="true">background-color: yellow;
color: rgb(224, 27, 36);</string>
                 </property>
                 <property name="text">
                  <string>*</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QPushButton" name="Button_atan">
                 <property name="text">
                  <string>atan</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_tan">
                 <property name="text">
                  <string>tan</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_sqrt">
                 <property name="styleSheet">
                  <string notr="true">background-color: yellow;
color: rgb(224, 27, 36);</string>
                 </property>
                 <property name="text">
                  <string>sqrt</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_add">
                 <property name="styleSheet">
                  <string notr="true">background-color: yellow;
color: rgb(224, 27, 36);</string>
                 </property>
                 <property name="text">
                  <string>+</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QPushButton" name="Button_log">
                 <property name="text">
                  <string>log</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_ln">
                 <property name="text">
                  <string>ln</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_x">
                 <property name="text">
                  <string>x</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_min">
                 <property name="styleSheet">
                  <string notr="true">background-color: yellow;
color: rgb(224, 27, 36);</string>
                 </property>
                 <property name="text">
                  <string>-</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="Button_BracketRight">
                 <property name="text">
                  <string>)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_BracketLeft">
                 <property name="text">
                  <string>(</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_C">
                 <property name="styleSheet">
                  <string notr="true">background-color: red</string>
                 </property>
                 <property name="text">
                  <string>C</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QPushButton" name="Button_9">
                 <property name="text">
                  <string>9</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_8">
                 <property name="text">
                  <string>8</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_7">
                 <property name="text">
                  <string>7</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QPushButton" name="Button_6">
                 <property name="text">
                  <string>6</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_5">
                 <property name="text">
                  <string>5</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_4">
                 <property name="text">
                  <string>4</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QPushButton" name="Button_3">
                 <property name="text">
                  <string>3</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_2">
                 <property name="text">
                  <string>2</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_1">
                 <property name="text">
                  <string>1</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QPushButton" name="Button_eq">
                 <property name="styleSheet">
                  <string notr="true">background-color: green</string>
                 </property>
                 <property name="text">
                  <string>=</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_comma">
                 <property name="text">
                  <string>.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Button_0">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
